### PR TITLE
Feature/133 mapbox activity

### PR DESCRIPTION
**Pull request by ToufiPF**
Brings the "MapboxActivity" to the app.

I created a TourDto, and modified ExperienceDto to support the parcelization of GeoPoint. I also added a new field in ExperienceDto that contains the documentId on firestore. This allows me to fetch the tour corresponding to the experience on Firestore.

I removed the Action bar (the title bar with "Pindex" written on it) to make more space for the map.

Tests are not completed yet, but comments are appreciated as always!

### CODE REVIEW

File: app/src/main/java/com/pindex/main/ui/activities/ExperienceActivity.kt [hunk 1]
```diff
@@ -89,6 +94,11 @@ class ExperienceActivity : AppCompatActivity() {
             )
             startTourView.visibility = View.VISIBLE
             startTourButton.setOnClickListener { onStartTour() }
+
```
Comment (lucasrlt) [inline app/src/main/java/com/pindex/main/ui/activities/ExperienceActivity.kt:97]: Great idea to fetch the tour data in background directly on the experience activity !

**Review by lucasrlt — APPROVED** [review]
Really impressive PR, it looks really good to me
Nice job on the parcelization for GeoPoints, and really smooth integration of Mapbox and permission handling in the activity

**Review by hadiyahya11 — APPROVED** [review]
Really good PR, the code is clean. Nice job

File: app/src/main/java/com/pindex/main/models/TourDto.kt [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    val featureCollection: FeatureCollection?
+        get() = geojson?.let { FeatureCollection.fromJson(it) }
+
+
+    @Suppress("UNCHECKED_CAST")
+    constructor(parcel: Parcel) : this(
+        startLocation = parcel.readGeoPoint(),
+        endLocation = parcel.readGeoPoint(),
+        goToStartContent =
+        parcel.readArrayList(ContentDto::class.java.classLoader) as ArrayList<ContentDto>?,
```
Comment (amomo25) [inline app/src/main/java/com/pindex/main/models/TourDto.kt:45]: Maybe add an indent at the beginning of the line for readability

**Review by amomo25 — APPROVED** [review]
Great PR, great job on parcelizing GeoPoints, I only saw an indent error. Other than that everything looks fine.

File: app/src/main/java/com/pindex/main/models/TourDto.kt (hunk 2 above)
Comment (ToufiPF) [reply app/src/main/java/com/pindex/main/models/TourDto.kt:45]: Actually I did put it, and Android studio removed it when I ran reformat code so...

Comment (amomo25) [reply app/src/main/java/com/pindex/main/models/TourDto.kt:45]: Ok, I don't think it's a big issue anyway

**Pull request merged**